common/src/main/java/com/wynntils/core/features/DebugFeature.java

```diff
@@ -4,20 +4,8 @@
  */
 package com.wynntils.core.features;
 
-import com.google.common.collect.ImmutableList;
-import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.features.properties.StartDisabled;
 
 /** Feature for debugging */
-public abstract class DebugFeature extends UserFeature {
-    @Override
-    protected void onInit(ImmutableList.Builder<Condition> conditions) {
-        conditions.add(new DevelopmentCondition());
-    }
-
-    public static class DevelopmentCondition extends Condition {
-        @Override
-        public void init() {
-            setSatisfied(WynntilsMod.isDevelopmentEnvironment());
-        }
-    }
-}
+@StartDisabled
```

Comment:
We can "inline" this class by just pushing @StartDisabled to the DebugFeature classes. I'm guessing you are saving that change to do separately? (I can do it as well)

Reply:
I did not want to do any changes to the features itself, just the backend.